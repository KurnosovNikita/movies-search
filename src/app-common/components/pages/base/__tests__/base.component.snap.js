import React from 'react';
import {shallow} from 'enzyme';
import Base from '../base.component';
import SearchPage from '../../../pages/search/search.component';

describe('Base component', () => {
    let props;
    beforeEach(() => {
        props = {
            data: <SearchPage />
        };
    });

    it('should render <Base> correctly', () => {
        const component = shallow(<Base {...props}/>);

        expect(component).toMatchSnapshot();
    });
});
