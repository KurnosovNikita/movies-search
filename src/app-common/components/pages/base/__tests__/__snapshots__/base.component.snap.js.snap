// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Base component should render <Base> correctly 1`] = `
<Fragment>
  <div
    className="wrapper"
  >
    <Search />
  </div>
  <Footer
    contextClassName="footer-content"
    footerClassName="footer"
  />
</Fragment>
`;
